tests/testProcessCcd.py

```diff
@@ -58,6 +58,7 @@ def setUp(self):
                      "isr.assembleCcd.setGain=False",
                      # This test uses CP-MasterCal calibration products
                      "-C", "%s/processCcdCpIsr.py" % configPath]
+        argsList.append('--doraise')
```

Comment:
Good point! Should I clarify in the commit?

Your guess is almost correct: previously, `ProcessCcdTask.parseAndRun` was failing without raising an exception, and so the subsequent tests would fail because the files the expected were not created, and thus it was really unclear where the problem was.